{"name": "test_vwo_login_positive", "status": "broken", "statusDetails": {"message": "AttributeError: 'TestLogin' object has no attribute 'name'", "trace": "self = <test_vwologin_pom.TestLogin object at 0x0000028B4DB055E0>\nsetup = <selenium.webdriver.chrome.webdriver.WebDriver (session=\"f4d5ad6704a6d01700e98b5c38298133\")>\n\n    @allure.epic(\"VWO Login Test\")\n    @allure.feature(\"TC#1 - VWO App positive Test\")\n    @pytest.mark.usefixtures(\"setup\")\n    @pytest.mark.positive\n    def test_vwo_login_positive(self, setup):\n        driver = setup\n        driver.get(self.base_url)\n        loginPage = LogInPage(driver)\n    \n>       loginPage.login_vwo(username=self.name, password=self.password)\nE       AttributeError: 'TestLogin' object has no attribute 'name'\n\ntest_vwologin_pom.py:36: AttributeError"}, "start": 1734938194526, "stop": 1734938197297, "uuid": "02a21582-54c1-4234-bb87-d27ff04622e1", "historyId": "9bef3ce1e555087702fcf864ae7a691e", "testCaseId": "9bef3ce1e555087702fcf864ae7a691e", "fullName": "tests.vwoLoginTests.pom.test_vwologin_pom.TestLogin#test_vwo_login_positive", "labels": [{"name": "epic", "value": "VWO Login Test"}, {"name": "feature", "value": "TC#1 - VWO App positive Test"}, {"name": "tag", "value": "positive"}, {"name": "tag", "value": "@pytest.mark.usefixtures('setup')"}, {"name": "parentSuite", "value": "tests.vwoLoginTests.pom"}, {"name": "suite", "value": "test_vwologin_pom"}, {"name": "subSuite", "value": "TestLogin"}, {"name": "host", "value": "JyotsnaSree"}, {"name": "thread", "value": "8316-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.vwoLoginTests.pom.test_vwologin_pom"}]}